<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_Ecran</class>
 <widget class="QMainWindow" name="ihm_Ecran">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Prepa</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="6" column="0" rowspan="5">
         <widget class="QPushButton" name="pB_Couleur2">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>110</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(252, 233, 79);</string>
          </property>
          <property name="text">
           <string>JAUNE</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3" rowspan="3" colspan="4">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="1" rowspan="2">
         <widget class="QLed" name="qLed_orange">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Binary Led</string>
          </property>
          <property name="whatsThis">
           <string>Led widget</string>
          </property>
          <property name="value">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QDoubleSpinBox" name="VBatt">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="maximum">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>23</pointsize>
           </font>
          </property>
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="11">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="6">
         <widget class="QPushButton" name="pB_Couleur1">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>110</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(173, 127, 168);</string>
          </property>
          <property name="text">
           <string>VIOLET</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="lbl_ModeFct">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>No Mode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="4">
         <widget class="QLed" name="qLed_green">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Binary Led</string>
          </property>
          <property name="whatsThis">
           <string>Led widget</string>
          </property>
          <property name="value">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="4">
         <widget class="QSpinBox" name="sb_ARG">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="5">
         <widget class="QSpinBox" name="sb_ARD">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="4">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ARG</string>
          </property>
         </widget>
        </item>
        <item row="9" column="5">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ARD</string>
          </property>
         </widget>
        </item>
        <item row="8" column="4">
         <widget class="QSpinBox" name="sb_AVG">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="5">
         <widget class="QSpinBox" name="sb_AVD">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>AVG</string>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>AVD</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Telemetres</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QLed" name="qLed_Obstacle">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Binary Led</string>
          </property>
          <property name="whatsThis">
           <string>Led widget</string>
          </property>
          <property name="value">
           <bool>true</bool>
          </property>
          <property name="onColor">
           <enum>QLed::Red</enum>
          </property>
          <property name="offColor">
           <enum>QLed::Grey</enum>
          </property>
          <property name="shape">
           <enum>QLed::Circle</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>match</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QDoubleSpinBox" name="sb_x">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-400.000000000000000</double>
          </property>
          <property name="maximum">
           <double>400.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="3">
         <widget class="QDoubleSpinBox" name="sb_y">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>-400.000000000000000</double>
          </property>
          <property name="maximum">
           <double>400.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="sb_teta">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>48</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>-400.000000000000000</double>
          </property>
          <property name="maximum">
           <double>400.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLed" name="qLed_conv">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Binary Led</string>
          </property>
          <property name="whatsThis">
           <string>Led widget</string>
          </property>
          <property name="value">
           <bool>false</bool>
          </property>
          <property name="onColor">
           <enum>QLed::Green</enum>
          </property>
          <property name="offColor">
           <enum>QLed::Grey</enum>
          </property>
          <property name="shape">
           <enum>QLed::Circle</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>CONVERGENCE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLed" name="qLed_obst">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Binary Led</string>
          </property>
          <property name="whatsThis">
           <string>Led widget</string>
          </property>
          <property name="value">
           <bool>false</bool>
          </property>
          <property name="onColor">
           <enum>QLed::Red</enum>
          </property>
          <property name="offColor">
           <enum>QLed::Grey</enum>
          </property>
          <property name="shape">
           <enum>QLed::Circle</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>OBSTACLE</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLed" name="qLed_bloc">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Binary Led</string>
          </property>
          <property name="whatsThis">
           <string>Led widget</string>
          </property>
          <property name="value">
           <bool>false</bool>
          </property>
          <property name="onColor">
           <enum>QLed::Orange</enum>
          </property>
          <property name="offColor">
           <enum>QLed::Grey</enum>
          </property>
          <property name="shape">
           <enum>QLed::Circle</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>BLOCAGE</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>fin</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="Q7seg" name="tps_dix">
          <property name="toolTip">
           <string>7 segment display</string>
          </property>
          <property name="whatsThis">
           <string>7 segment display</string>
          </property>
          <property name="num" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Q7seg" name="tps_unit">
          <property name="toolTip">
           <string>7 segment display</string>
          </property>
          <property name="whatsThis">
           <string>7 segment display</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_Score">
       <attribute name="title">
        <string>SCORE</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="Q7seg" name="score_cent">
          <property name="toolTip">
           <string>7 segment display</string>
          </property>
          <property name="whatsThis">
           <string>7 segment display</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Q7seg" name="score_dix">
          <property name="toolTip">
           <string>7 segment display</string>
          </property>
          <property name="whatsThis">
           <string>7 segment display</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Q7seg" name="score_unit">
          <property name="toolTip">
           <string>7 segment display</string>
          </property>
          <property name="whatsThis">
           <string>7 segment display</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>CONNECT</string>
       </attribute>
       <widget class="QPushButton" name="pB_connect">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>300</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="text">
         <string>OPEN</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pB_close">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>110</y>
          <width>300</width>
          <height>60</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>60</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>48</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CLOSE</string>
        </property>
       </widget>
       <widget class="QLed" name="isOpen">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>50</y>
          <width>50</width>
          <height>50</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Binary Led</string>
        </property>
        <property name="whatsThis">
         <string>Led widget</string>
        </property>
        <property name="value">
         <bool>true</bool>
        </property>
        <property name="onColor">
         <enum>QLed::Green</enum>
        </property>
        <property name="offColor">
         <enum>QLed::Grey</enum>
        </property>
        <property name="shape">
         <enum>QLed::Circle</enum>
        </property>
       </widget>
       <widget class="QLed" name="isClose">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>110</y>
          <width>50</width>
          <height>50</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Binary Led</string>
        </property>
        <property name="whatsThis">
         <string>Led widget</string>
        </property>
        <property name="value">
         <bool>true</bool>
        </property>
        <property name="onColor">
         <enum>QLed::Red</enum>
        </property>
        <property name="offColor">
         <enum>QLed::Grey</enum>
        </property>
        <property name="shape">
         <enum>QLed::Circle</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Q7seg</class>
   <extends>QWidget</extends>
   <header>q7seg.h</header>
  </customwidget>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>qled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
